server/src/main/java/com/linecorp/centraldogma/server/CentralDogma.java

```diff
@@ -383,8 +375,35 @@ private CommandExecutor startCommandExecutor(
         return executor;
     }
 
+    @Nullable
+    private SessionManager initializeSessionManager() {
```

Comment:
Could add `throws Exception` and remove all catches?

Reply:
Fixed.